stytch/config/config.go

```diff
@@ -1,6 +1,6 @@
 package config
 
-const APIVersion = "5.15.0"
+const APIVersion = "5.15.1"
```

Comment:
I think this deserves a minor version bump
```suggestion
const APIVersion = "5.16.0"
```